### PR TITLE
Deal correctly with favicon

### LINKED ISSUE #810: Deal correctly with favicon link
We accidentally extract `<link rel="shortcut icon" href=".../favicon.ico">` as if it were a stylesheet and transform the contents into inline style tags. We need to add logic to treat the icon as an image instead.

See https://github.com/kiwix/kiwix-js/issues/790#issuecomment-1024900020 for a screenshot of the issue in console log.

### PR DESCRIPTION
**Pull request by Jaifroid**
Fixes #810.

### CODE REVIEW

**Comment by Jaifroid** [comment]
I've also de-jQueried the `loadCSSJquery` function as it does weird things to the link node (puts it inside an array unnecessarily).

**Comment by Jaifroid** [comment]
I've also de-jQueried the `loadCSSJquery` function as it does weird things to the link node (puts it inside an array unnecessarily).

**Review by mossroy — APPROVED** [review]
The code looks good.
Did not test, though

File: www/js/app.js [hunk 1]
```diff
@@ -1716,28 +1716,29 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
 
             var cssCount = 0;
             var cssFulfilled = 0;
-            $('#articleContent').contents().find('link[data-kiwixurl]').each(function () {
+            var iframe = iframeArticleContent.contentDocument;
+            Array.prototype.slice.call(iframe.querySelectorAll('link[data-kiwixurl]')).forEach(function (link) {
```
[the remark this answers is not in the document]
Comment (Jaifroid) [reply www/js/app.js:1720]: Yes, it is supported if it's an array, but not a node. That's the reason for using Array.prototype.slice.call -- using the Array.forEach method and "applying" it to the node list.

**Pull request merged**